BoxContentSDK/BoxContentSDK/Views/BOXUserAvatarImageView.m

```diff
@@ -125,6 +125,15 @@ - (void)renderRealUserAvatarFromRemote
     }];
 }
 
+- (void)refresh
+{
+    if (self.user.modelID.length > 0) {
+        [self.request cancel];
+        self.request = [self.client userAvatarRequestWithID:self.user.modelID type:BOXAvatarTypeLarge];
+        [self renderRealUserAvatarFromRemote];
```

Comment:
How do you feel about this pattern? My gut says that having something like `[self renderRealUserAvatarFrom:request]` is better.

Reply:
done